cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -368,8 +368,8 @@ func ValidateHostPort(endpoint string, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
-// ValidateIPNetFromString validates network portion of ip address
-func ValidateIPNetFromString(subnet string, minAddrs int64, fldPath *field.Path) field.ErrorList {
+// ValidateServiceSubnetFromString validates network portion of service subnet
+func ValidateServiceSubnetFromString(subnet string, minAddrs int64, fldPath *field.Path) field.ErrorList {
```

Comment:
For now we don't have feature-gates used in kubeadm validation, I'd expect that`ValidateServiceSubnetFromString` and `ValidatePodSubnetsFromString` work as expected in both cases

cc @timothysc @neolit123

Reply:
will comment on feature gates in the bellow thread under ValidateNetworking.